Project-I-Social-Card/social-card/src/components/HeaderComponents/ImageThumbnail.js

```diff
@@ -1,2 +1,9 @@
 import React from 'react';
 import './Header.css';
+
+
+const ImageThumbnail= () => {
+    return <img class= 'thumbnail' src='https://tk-assets.lambdaschool.com/1c1b7262-cf23-4a9f-90b6-da0d3c74a5c6_lambdacrest.png'></img>
```

Comment:
img elements are self-closing tags in JSX. You can also add an alt attribute for better accessibility and to default to in case the image fails to load.
https://facebook.github.io/create-react-app/docs/adding-images-fonts-and-files

Reply:
*class* should be *className*